meson.build

```diff
@@ -257,7 +257,7 @@ foreach tuple : _completiondirs
   if completiondir != 'no'
     foreach comp : _completions
       install_data(
-        files(ins_path.format(comp)),
+        files('shell-completion/@0@/'.format(tuple[0]) + ins_path.format(comp)),
```

Comment:
we should prefer join_paths() for path building. Or event better... use / operator that is available since meson 0.49

Reply:
Ack, switched to `/` with prep commit and updated the patches to use `/`.